### PR TITLE
New json logging

### PR DESCRIPTION
**Pull request by DanielElisenberg**
Tested with system-test locally

### CODE REVIEW

File: job_executor/config/log.py [hunk 1]
```diff
@@ -1,114 +1,98 @@
...
-
-
-def configure_worker_logger(queue: Queue, job_id: str):
-    queue_handler = logging.handlers.QueueHandler(queue)
-    queue_handler.setLevel(logging.INFO)
-    # use default formatter here so that the JSON formatter will be
-    # applied when reading from queue
-    queue_handler.formatter = logging.Formatter()
+    def format(self, record: logging.LogRecord) -> str:
+        return f"{self.job_id}: {record.msg}"
```
Comment (pawbu) [inline job_executor/config/log.py:56]: If the worker returns only the message, and the main thread does the rest of the logging, can the timestamp be a bit off here?

Comment (DanielElisenberg) [reply job_executor/config/log.py:56]: Hmm I did not consider this. This actually extends to error stacks also, I'll have a second look 💪🏻

File: job_executor/config/log.py [hunk 2]
```diff
@@ -1,114 +1,105 @@
...
-
-def configure_worker_logger(queue: Queue, job_id: str):
-    queue_handler = logging.handlers.QueueHandler(queue)
-    queue_handler.setLevel(logging.INFO)
-    # use default formatter here so that the JSON formatter will be
-    # applied when reading from queue
-    queue_handler.formatter = logging.Formatter()
+    def format(self, record: logging.LogRecord) -> str:
+        record.msg = f"{self.job_id}: {record.msg}"
+        return record
```
Comment (pawbu) [inline job_executor/config/log.py:63]: The `format` method should return a `str` ([ref](https://docs.python.org/3/library/logging.html#logging.Formatter.format)). Maybe use a filter like the previous version of the code?

**Review by pawbu — APPROVED** [review]
Looks good 🚀

**Pull request merged**